lib/driver.rb

```diff
@@ -2,26 +2,28 @@
 require_relative './TestScriptEngine'
 require './TestScriptRunnable' # TODO: Remove
 
-test_server_url = 'http://server.fire.ly'
-testscript_path = '../TestScripts'
-testscript_file = nil
+endpoints = []
+directory_path = '../TestScripts'
+file_name = nil
 
 parameters = ARGV
 parameters.each do |parameter|
-  if parameter.start_with?('http')
-    test_server_url = parameter
-  elsif parameter.include?('.json')
-    testscript_file = parameter
+  if parameter.include?('.json') || parameter.include?('.xml')
```

Comment:
I think we should leave this out. I think the best way to do this would be to enter endpoints during execution -- maybe add some code there to prompt the user? And allow them to not provide any, and just use whatever is specified in the base TestScript

Reply:
I am slow to respond this. In a separate conversation in Slack we agreed that the engine consumes endpoints from arguments.